test/jdk/java/foreign/loaderLookup/TestSymbolLookupFindOrThrow.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/*
+ * @test
+ * @run junit/othervm --enable-native-access=ALL-UNNAMED TestSymbolLookupFindOrThrow
+ */
+
+import java.lang.foreign.MemorySegment;
+import java.lang.foreign.SymbolLookup;
+import java.util.NoSuchElementException;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.Assert.*;
+
+final class TestSymbolLookupFindOrThrow {
+
+    static {
+        System.loadLibrary("Foo");
+    }
+
+    @Test
+    void findOrThrow() {
+        MemorySegment symbol = SymbolLookup.loaderLookup().findOrThrow("foo");
+        Assertions.assertNotEquals(0, symbol.address());
+    }
+
+    @Test
+    void findOrThrowNotFound() {
+        assertThrows(NoSuchElementException.class, () ->
+                SymbolLookup.loaderLookup().findOrThrow("bar"));
+    }
+
+    @Test
+    void findOrThrowNullArg() {
+        assertThrows(NullPointerException.class, () ->
```

Comment:
I believe this should already be checked by the TestNulls test - which is a test that automatically checks that _all_ API methods handle nulls accordingly. Please check that, and maybe remove this?